package.xml

```diff
@@ -2,29 +2,31 @@
   <name>srdfdom</name>
   <version>0.3.1</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
-  <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
-  <maintainer email="isucan@willowgarage.com">Ioan Sucan</maintainer>
+  <maintainer email="isucan@gmail.com">Ioan Sucan</maintainer>
```

Comment:
he suggested he be removed "from all instances" of package.xmls with him as a maintainer. I think we should also start a separate discussion on Discourse to find new active maintainers for all the packages - want to start that?